org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -12,6 +12,16 @@
  *******************************************************************************/
 package org.eclipse.lsp4e;
 
+import org.eclipse.core.resources.IProject;
```

Comment:
it looks like organize imports is configured different than by default. Could you change it so that the PR does not change the order of the import in all the modified files?